react/features/overlay/components/PageReloadOverlay.js

```diff
@@ -0,0 +1,298 @@
+             * darker otherwise.
+             * @type {boolean}
+             */
+            isLightOverlay,
+
+            /**
+             * The translation key for the title of the overlay
+             * @type {string}
+             */
+            message,
+
+            /**
+             * How long the overlay dialog will be
+             * displayed, before the conference will be reloaded.
+             * @type {number}
+             */
+            timeoutSeconds,
+
+            /**
+             * The translation key for the title of the overlay
+             * @type {string}
+             */
+            title
+        };
+    }
+
+    /**
+     * Renders the button for relaod the page if necessary.
+     *
+     * @returns {ReactElement|null}
+     * @private
+     */
+    _renderButton() {
+        if (this.props.isNetworkFailure) {
+            const cName = 'button-control button-control_primary '
+                + 'button-control_center';
+
+            /* eslint-disable react/jsx-handler-names */
+
+            return (
+                <button
+                    className = { cName }
+                    data-i18n = 'dialog.reconnectNow'
+                    id = 'reconnectNow'
+                    onClick = { this._reconnectNow } />
+            );
+        }
+
+        return null;
+    }
+
+    /**
+     * Constructs overlay body with the warning message and count down towards
+     * the conference reload.
+     *
+     * @returns {ReactElement|null}
+     * @override
+     * @protected
+     */
+    _renderOverlayContent() {
+
+        /* eslint-disable react/jsx-handler-names */
+
+        return (
+            <div className = 'inlay'>
+                <span
+                    className = 'reload_overlay_title'
+                    data-i18n = { this.state.title } />
+                <span
+                    className = 'reload_overlay_text'
+                    data-i18n = { this.state.message } />
+                <ReloadTimer
+                    end = { 0 }
+                    interval = { 1 }
+                    onFinish = { this._reconnectNow }
+                    start = { this.state.timeoutSeconds }
+                    step = { -1 } />
+                { this._renderButton() }
+            </div>
+        );
+    }
+
+    /**
+     * This method is executed when comonent is mounted.
+     *
+     * @inheritdoc
+     * @returns {void}
+     */
+    componentDidMount() {
+        super.componentDidMount();
+
+        // FIXME (CallStats - issue) this event will not make it to
+        // the CallStats, because the log queue is not flushed, before
+        // "fabric terminated" is sent to the backed
+        // FIXME: We should dispatch action for this
+        APP.conference.logEvent('page.reload', undefined /* value */,
+            this.props.reason /* label */);
+        logger.info(`The conference will be reloaded after
+            ${this.state.timeoutSeconds} seconds.`);
```

Comment:
The template literal above does take the newline and all the spaces from the beginning of the second line into the resulting string. Which is not a real problem in this case, one just ends up with a funny looking info log line.